### PR TITLE
Adding support for Hotel booking API

### PR DESCRIPTION
**Pull request by akshitsingla**
## Changes for this pull request
#41  Adding support for Hotel booking API

### CODE REVIEW

File: src/amadeus/namespaces/booking.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * @protected
+ */
+class Booking {
+  constructor(client) {
+    this.client    = client;
+    this.hotelBookings = new HotelBookings(client);
+  }
+}
+
+export default Booking;
```
Comment (tsolakoua) [inline src/amadeus/namespaces/booking.js]: should the newlines at the end of the files be added? I am not sure about JS.

File: src/amadeus/namespaces/booking/hotel_bookings.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+   *  'payments': []
+   * });
+   * ```
+   */
+  post(params = {}) {
+    return this.client.post('/v1/booking/hotel-bookings', params);
+  }
+}
+
+export default HotelBookings;
```
Comment (tsolakoua) [inline src/amadeus/namespaces/booking/hotel_bookings.js]: here as well

File: src/amadeus/namespaces/booking/hotel_bookings.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    this.client = client;
+  }
+
+  /**
+   * To book the offer retrieved from Hotel Shopping API.
+   *
+   * @param {Object} params
+   * @return {Promise.<Response,ResponseError>} a Promise
+   *
+   * To book the hotel offer with ID 'XXX' with guests & payaments info
```
Comment (tsolakoua) [inline src/amadeus/namespaces/booking/hotel_bookings.js]: payaments typo

File: src/amadeus/namespaces/booking.js (hunk 1 above)
Comment (akshitsingla) [reply src/amadeus/namespaces/booking.js]: Nope, we don't keep extra lines in the end. Also, none of the previous files has it.

File: src/amadeus/namespaces/booking/hotel_bookings.js (hunk 2 above)
Comment (akshitsingla) [reply src/amadeus/namespaces/booking/hotel_bookings.js]: Same here.

File: src/amadeus/namespaces/booking/hotel_bookings.js (hunk 3 above)
Comment (akshitsingla) [reply src/amadeus/namespaces/booking/hotel_bookings.js]: Fixed.

**Pull request merged**